### PR TITLE
L2/L1 synths

### PR DESCRIPTION
**Pull request by Rickk137**
https://github.com/Synthetixio/synthetix-website/issues/174
https://www.figma.com/file/ZchEcPJuaIowqOLdxG4o2B/SynthsPage?node-id=2%3A171

### CODE REVIEW

File: src/sections/synths/SynthsInfo.tsx [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
+	const [showL2, setShowL2] = useState(true);
 	const [synthCategory, setSynthCategory] = useState('all');
 	const filteredSynths: Record<string, ReactNode[]> = {
 		all: [],
 	};
 
+	const { synths, exchangeInfo, dictionarySynthStatus } = useMemo(
+		() => (showL2 ? l2 : l1),
+		[showL2, l1, l2]
+	);
```
Comment (bachstatter) [inline src/sections/synths/SynthsInfo.tsx:20]: Dont think we need a memo here. It's should be faster (and simpler) to just 
`const { synths, exchangeInfo, dictionarySynthStatus } = showL2 ? l2 : l1`

File: tests/e2e/specs/synths-page.spec.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 describe('Synths Page', () => {
 	it.only('should have at least 3 tabs', () => {
 		cy.visit('http://localhost:3000/synths');
-		cy.get('button[data-test-id="tab-categories"]').should('have.length.gt', 3);
```
Comment (bachstatter) [inline tests/e2e/specs/synths-page.spec.js:4]: Looks like the e2e test still failed :(

**Review by bachstatter — COMMENTED** [review]
Nice work! Some small comments and then need to figure out the e2e test failure too

File: pages/synths.tsx [hunk 3]
```diff
@@ -11,46 +11,79 @@ import exchangeInfoQuery, { Fees, Rates } from 'src/queries/exchangeInfo/useExch
...
+};
+
+export async function getStaticProps() {
+	const [
+		tokenListResponse,
+		{ exchangeInfoL1, exchangeInfoL2 },
+		{ synths: synthsL1, dictionarySynthStatus: dictionarySynthStatusL1 },
+		{ synths: synthsL2, dictionarySynthStatus: dictionarySynthStatusL2 },
+	] = await Promise.all([
+		axios.get<TokenListResponse>('https://synths.snx.eth.link'),
```
Comment (fritzschoff) [inline pages/synths.tsx:42]: I guess we can get rid of that? cause we get every information form chain? The link only returns L1 synths anyways, right?

File: src/components/Checkbox/Checkbox.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+
+export interface CheckboxProps {
+	label?: string | React.ReactNode;
+	onChange: (checked: boolean) => void;
+	checked: boolean;
+}
+
+export const Checkbox: React.FC<CheckboxProps> = ({ label, onChange, checked, ...props }) => {
```
Comment (fritzschoff) [inline src/components/Checkbox/Checkbox.tsx:9]: nice components. We can port that over to the ui lib. Pretty sure we are gonna use that in the staking app and in the v3gm interface as well. 
https://github.com/synthetixio/ui
Can you make a TODO here and create an issue in this repo here so we don't forget

Comment (Rickk137) [reply src/components/Checkbox/Checkbox.tsx:9]: created the issue:
https://github.com/Synthetixio/ui/issues/28

File: pages/synths.tsx (hunk 3 above)
Comment (Rickk137) [reply pages/synths.tsx:42]: well we are getting the "logoURI" property for SynthCard. I don't think that data can is onChain.

Comment (fritzschoff) [reply pages/synths.tsx:42]: can't we do something like that here:
```
	<StyledSynthImage
				src={`https://raw.githubusercontent.com/Synthetixio/synthetix-assets/v2.0.13/synths/${name}.svg`}
			/>
```
https://github.com/Synthetixio/synthetix-website/blob/staging/src/components/FutureSynthCard/index.tsx#L16
all the svgs are stored in that repo

File: src/sections/synths/SynthsInfo.tsx [hunk 5]
```diff
@@ -72,18 +94,25 @@ const Cards = styled.div`
...
+	padding-left: 16px;
+`;
+
+const Filters = styled.div`
+	display: flex;
+	margin-bottom: 40px;
+	justify-content: start;
+	align-items: center;
+	flex-wrap: wrap;
+	gap: 28px;
```
Comment (fritzschoff) [inline src/sections/synths/SynthsInfo.tsx:110]: brave but lets try
https://caniuse.com/?search=gap

**Review by fritzschoff — APPROVED** [review]

**Pull request merged**